notebooks/data-sources/Bugzilla/bugzilla_EDA.ipynb

```diff
@@ -0,0 +1,3552 @@
+       "10    NaN\n",
+       "11    NaN\n",
+       "12    NaN\n",
+       "13    NaN\n",
+       "14    NaN\n",
+       "Name: fixed_in, dtype: object"
+      ]
+     },
+     "execution_count": 101,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "bugs_df[\"fixed_in\"][:15]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 102,
+   "id": "04c00ac4",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "array([nan, 'v0.1.10', 'podman-1.9.3-1.rhaos4.6.el8',\n",
+       "       'OCP-Metal-v1.0.18.2', 'OCP-Metal-v1.0.9.5',\n",
+       "       'milei@redhat.com , annair@redhat.com', '4.8',\n",
+       "       'annair@redhat.com, milei@redhat.com',\n",
+       "       'virt-cdi-importer 2.6.0-15', 'jkaur@redhat.com',\n",
+       "       'OCP-Metal-v1.0.18.1', 'cri-o-1.19.0-62.rhaos4.6.git10c7a86.el8',\n",
+       "       'OCP-Metal-V1.0.17.3', 'assisted-ui-lib v0.0.13-wizard',\n",
+       "       '4.7.0-0.nightly-2020-12-17-001141',\n",
+       "       'cri-o-1.20.2, openshift 4.7.4', '4.6.4', 'facet-lib v1.4.9',\n",
+       "       'runc-1.0.0-82.rhaos4.6.git086e841.el8',\n",
+       "       'runc-1.0.0-67.rc10.rhaos4.3.el7', '2.5.0',\n",
+       "       'podman-1.6.4-11.rhaos4.3.el8',\n",
+       "       'runc-1.0.0-81.rhaos4.6.git5b757d4', 'OCP-Metal-v1.0.12.1'],\n",
+       "      dtype=object)"
+      ]
+     },
+     "execution_count": 102,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "bugs_df[\"fixed_in\"].unique()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ebf4683a",
+   "metadata": {},
+   "source": [
+    "The `fixed_in` field seems to indicate the software version the bug was fixed in. However, it doesn't seem to be applicable to all bugs as some bugs may still be open and not yet resolved."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "fff260e0",
+   "metadata": {},
+   "source": [
+    "## Contact Metadata\n",
+    "\n",
+    "These fields contain information for people responsible for QA, creation of bug, etc. These are not useful for the initial EDA.\n",
+    "\n",
+    "- `docs_contact_value` and `qa_contact`: The people responsible for contacting and fixing the bug.\n",
+    "- `creator`: The person who created the bug.\n",
+    "- `assigned_to`: The person responsible for fixing the bug.\n",
+    "- `cc`: The mailing list subscribed to get updates for a bug."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2e7f636f",
+   "metadata": {},
+   "source": [
+    "## Non Useful Metadata\n",
+    "\n",
+    "These fields mostly had either the same value or empty. Therefore, these are not useful for our analysis.\n",
+    "\n",
+    "- `summary`: The bug summary is a short sentence which succinctly describes what the bug is about.\n",
+    "- `is_open`: The bug is_open field returns a boolean value indicating if the bug is currently open or not.\n",
```

Comment:
It unclear to me why these two field are not valuable? I would think we would want to know if an bug is open or not and also have a succinct summary of it.

Reply:
I think, and @hemajv @aakankshaduggal can correct me here, the initial thought was that `is_open` is redundant (or less useful) coz we already have `status`, and `summary` is less useful coz we already have the whole `description`.

But now that you mention it, I think it might be valuable to include these as well

Reply:
Yes @chauhankaranraj , we dropped some of these fields due to redundancy. But we can definitely include these. :+1:

Reply:
+1 yes, if its due to redundancy you could note that as well in the reason for dropping it.